codegen_java/templates/libraries/okhttp-gson/api.mustache

```diff
@@ -0,0 +1,550 @@
+{{>licenseInfo}}
```

Comment:
I mustache you - how much of this is generated/hand-written? Is there a reference page you can add in as a comment?

Reply:
I wrote it all, on a stone tablet first then I scanned it.

But seriously, I surrounded the blocks I customized with comments but I will add a comment to the top

Reply:
Am picturing: https://www.youtube.com/watch?v=oZA2mBntrHk

Reply:
Ah ok - I got scared off because it looked a lot like ctmpl and that does me a frighten.